app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  belongs_to :user, required: false
```

Comment:
Great work practising how to use associations 👏🏿 
However, Since you have only one model User, you don't need an association with its self.

- [ ] Kindly remove the belongs to user. This is because there is no need of creating an association of a user with a user. You can only use this when you referencing to another model. Which is not required in this project